gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/ProductDependency.groovy

```diff
@@ -15,8 +15,8 @@ class ProductDependency {
 
     ProductDependency() {}
 
-    ProductDependency(String productGroup, String productName, @Nullable String minimumVersion,
-                      @Nullable String maximumVersion, @Nullable String recommendedVersion) {
+    ProductDependency(String productGroup, String productName, String minimumVersion,
```

Comment:
would expecting something to change in the code if these are no longer optional?